### PR TITLE
Final capstone

### PR DESCRIPTION
**Pull request by Eccb7**
Project requirements
 - You should personalize the content of your page. Choose a topic that is different than the one in the original design.

 You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

 - Colors.
 - Typography: font face, size, and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
 - Layout: composition and space between elements.
 - The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

 - You can use a CSS framework (for example Bootstrap) for styling, if you want to.

You should build only these 2 pages:

 - The home page.
  -The about page.
 Each of these pages should have versions for 2 different screen sizes:

 - Mobile: up to 768px wide.
 - Desktop: 768px or wider.
Interactions

Links

-  The home page should have a link in the menu to the about page.
-  The logo in the header links to the home page.
Mobile menu

 - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

 There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc).

 - The mobile menu should have a close (X) button that closes the menu.

-  Dynamic page

  - The section "Featured speakers" should be created dynamically in JavaScript.

 You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

 The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that I give appropriate credit to the author. Therefore, I do it in the README of my project.

**Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -446,7 +446,7 @@ body {
 #footer {
   display: flex;
   justify-content: space-between;
-  background-color: #dedede;
+  background-color: #f6f4f4;
   padding: 0;
 }
```
Comment (V-Blaze) [inline style.css]: - [x] You have done great so far but I think there are some improvements you need to make in the hero section. :+1: :100: 
- - As part of the requirement is to replicate the given design as much as possible, I think you need to adjust the text sizing and also the spacing and margin around the content. I suggest you look at the design and adjust this section accordingly :+1: :smile: 

- - Your Current Design:
Screenshot from 2023-05-31 16-40-14

- - Provided Design:
Selection_052


- [x] I think you mixed the styling for the mobile and desktop wrongly. In the mobile screen,  The `Partners` and the `Footer` is not visible in the provided design, but this is the opposite is your own design. I'll suggest you adjust these styling appropriately. :smile: :+1: 

VIDEO FOR REFERENCE:

https://github.com/Eccb7/Capstone_Project-1/assets/107632780/443650c3-38c1-47d3-bbd0-8c510b3d255b

- - Looking at the video, you can see that you showed what is supposed to be hidden in mobile, and then hide what is supposed to be visible bith in Home page and the About page.

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Eccb7 , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Meaningful commit message✔
- Correct Github flow👋 
-  Well structured code✔

 ## Required Changes ♻️
- [ ] You left an unused code as a comment in your `script.js` line 61 to 64. THis is against the best practices as it makes your code base look unprofessional. I'll suggest you remove this. :+1: :100:  
- [ ] Pull request summary should be a description of the features you implemented on the current branch and not just pasting the project requirement that was provided, I'll suggest you edit the pull request description and list the actions and features you actually added in this branch :+1: :smile: 

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Eccb7) [reply style.css]: I thought am not to adhere to everything but just personalize my work and make it distinct?

File: style.css [hunk 2]
```diff
@@ -392,61 +401,83 @@ body {
...
+.partners1 {
+  background-color: #272a31;
+  color: #dedede;
+  padding-bottom: 20px;
+}
+
+.partners1 h3 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (OyePriscilla) [inline style.css:426]: - [ ] Please remove the partner and footer sections from the homepage mobile view as suggested earlier. I understand your concept to be dynamic, but all the same let's try and keep to the given template. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Eccb7 👏, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Built two pages app ✔️ 
App featured section is dynamic with the use of JS ✔️ 

 ## Required Changes ♻️
**Kindly remove the commented code that is in your script.js file, this will make your codebase cleaner. ℹ️** 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -392,61 +391,90 @@ body {
...
+  background-color: #f6f4f4;
+  padding: 0;
+}
+
+#footer2 {
+  display: flex;
+  justify-content: space-between;
+  background-color: #f6f4f4;
   padding: 0;
 }
```
Comment (1GORDON) [inline style.css:502]: Great job @Eccb7 🤝

- [ ] As stated by the previous reviewer, kindly remove the footer from your mobile view of the home page.
- Your design
Your mobile footer
- Given design
Given mobile footer

File: about.html [hunk 4]
```diff
@@ -1,165 +1,159 @@
...
+      <nav>
+        <div>
+          <div class="mobile-menu">
+            <a href="#" class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </a>
+          </div>
+          <ul class="nav-list">
```
Comment (1GORDON) [inline about.html:44]: Great design @Eccb7 🤝

- [ ] [Optional] Kindly ensure there is a link to direct the user to the Home page from the mobile menu of the About section. Both mobile menus have an About button. Kindly ensure the mobile menu of the About section has the `Home` button instead.
About

File: index.html [hunk 5]
```diff
@@ -105,7 +105,7 @@ <h4>Math Ignite</h4>
     <section id="speakers">
         <div id="featured-speakers" class="speaker-card"></div>
```
Comment (1GORDON) [inline index.html:106]: Great design @Eccb7 🤝

- [ ] [Optional] I would recommend you implement the `more` button as given in the design guidelines.
More button

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Eccb7 ✋

Good job so far!

## Highlights!
- Followed GitHub Flow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Eccb7 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed GitHub Flow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**